### ISSUE TITLE
Problem in using module with node v7.4.0 (npm v4.0.5)

### ISSUE THREAD

**Issue by dimikhan** [laverdet/node-fibers#333]
I can use this module with node v4.1.2 and run my application. But the same application doesn't run with node v7.4.0. The error is as below. 

**c:\workspace_PL\PL_PACreation>node app.js
## There is an issue with `node-fibers` ##
`c:\workspace_PL\PL_PACreation\node_modules\synchronize\node_modules\fibers\bin\win32-x64-51\fibers.node` is missing.

Try running this to fix the issue: C:\Program Files\nodejs\node.exe c:\workspace_PL\PL_PACreation\node_modules\synchronize\node_modules\fibers/build
c:\workspace_PL\PL_PACreation\node_modules\synchronize\node_modules\fibers\fibers.js:20
        throw new Error('Missing binary. See message above.');
        ^

Error: Missing binary. See message above.
    at Object.<anonymous> (c:\workspace_PL\PL_PACreation\node_modules\synchronize\node_modules\fibers\fibers.js:20:8)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (c:\workspace_PL\PL_PACreation\node_modules\synchronize\sync.js:3:13)
    at Module._compile (module.js:571:32)

c:\workspace_PL\PL_PACreation>cd ..
c:\workspace_PL>cd test
c:\workspace_PL\test>npm install synchronize
> fibers@1.0.15 install c:\workspace_PL\test\node_modules\fibers
> node build.js || nodejs build.js**

**Comment by dimikhan** [comment]
I saw, the reason for this error is that when it runs with node v4.1.2 it expects the fibers.node in fibers\bin\win32-x64-46 and when it runs with node v7.4.0 it expects the fibers.node in fibers\bin\win32-x64-51.
When I installed synchronize module, the dependent fibers module did not get installed with win32-x64-51 folder (it has only till ***-48). Not sure if it is a problem with fibers installation or I am installing it incorrectly.
